### PR TITLE
Add Terraform Support

### PR DESCRIPTION
**Pull request by prateepb**
As discussed in #269, I've added Terraform support for EC2. This could easily be extended to support other providers

### CODE REVIEW

File: cloud/terraform/tpot.conf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# tpot configuration file
+# myCONF_TPOT_FLAVOR=[STANDARD, SENSOR, INDUSTRIAL, COLLECTOR, NEXTGEN, LEGACY]
```
Comment (t3chn0m4g3) [inline cloud/terraform/tpot.conf]: Just noticed there is no `LEGACY` edition any longer. Can you adjust this please?

**Comment by TheHADILP** [comment]
Looking good 👍
Maybe you can add Terraform as a bullet point in the main README (under Table of Contents --> Installation --> Cloud Deployments) and write some sentences and link to the subfolder.

File: cloud/terraform/tpot.conf (hunk 1 above)
Comment (prateepb) [reply cloud/terraform/tpot.conf]: Sure. I used [tpot.conf.dist](https://github.com/dtag-dev-sec/tpotce/blob/master/iso/installer/tpot.conf.dist) as an example though. I guess this should be changed as well?

Is this correct?

```
# tpot configuration file
# myCONF_TPOT_FLAVOR=[STANDARD, SENSOR, INDUSTRIAL, COLLECTOR, NEXTGEN]
myCONF_TPOT_FLAVOR='STANDARD'
myCONF_WEB_USER='webuser'
myCONF_WEB_PW='w3b$ecret'
```

Comment (t3chn0m4g3) [reply cloud/terraform/tpot.conf]: Thank you, missed that and will take care of it :)

Comment (prateepb) [reply cloud/terraform/tpot.conf]: No problem. I've updated on my side as well.

**Comment by TheHADILP** [comment]
Awesome! 🎉

**Comment by t3chn0m4g3** [comment]
Thanks @prateepb @TheHADILP

**Pull request merged**